metrics/exporter.go

```diff
@@ -121,6 +121,12 @@ func isNewExporterRequired(newConfig *metricsConfig) bool {
 		return true
 	}
 
+	// If the OpenCensus address has changed, restart the collector.
```

Comment:
restart the exporter? In bring-your-own-collector-service world, I don't think Knative should have the ability to or be responsible for restarting OpenCensus Service agents.

Reply:
Yes, this was supposed to be exporter.